src/fragments/lib/graphqlapi/js/setup/connect.mdx

```diff
@@ -341,8 +353,9 @@ In GraphQL, mutations write data to the API and are used to create, update, or d
 [comment]: # (EXPANDER SECTION that covers GraphQL basics, mutations, and why this is helpful—can be a short summary with recommended links.)
   
 [comment]: # (EXPANDER end)
+[comment]: # (RENE, what text are we planning to use within the "Learn more about GraphQL mutations" expander? We're missing a high-level overview of the basics and the benefits of mutations.)
 
-Here are some examples of creating, updating, and deleting items using the Amplify GraphQL client.
+Here are some examples of how you can create, update, and delete items using the Amplify GraphQL client.
 
```

Comment:
Note here that I thoguht an expander seemed like too much to remind them of the diff b/w mutations and queries. I added the brief review directly into this intro. We can change it back to something more robust if needed.